server/src/modules/mail/mailFunctions.ts

```diff
@@ -44,11 +44,11 @@ export const sendPublishedMailToStructureMembers = async (
   lien: string,
   dispositifId: Dispositif["_id"],
 ) =>
-  membres.map((membre) => {
+ Promise.all(membres.map(async (membre) => {
     logger.info("[sendPublishedMailToStructureMembers] send mail to membre", {
       membreId: membre._id,
     });
-    return sendPublishedFicheMailToStructureMembersService({
+    await sendPublishedFicheMailToStructureMembersService({
```

Comment:
```suggestion
    return sendPublishedFicheMailToStructureMembersService({
```

Reply:
Thanks @revolunet!  I simplified somewhat using the `void` operator.